designsafe/apps/data/views/base.py

```diff
@@ -199,12 +199,27 @@ def get_context_data(self, **kwargs):
         elif pub.project.to_dict().get('doi') != None: #This is for older publications
             context['doi'] = pub.project.doi
         context['keywords'] = pub.project.value.keywords.split(',')
-        context['authors'] = [{
-            'full_name': '{last_name}, {first_name}'.format(
-                last_name=user['last_name'], first_name=user['first_name']
-            ),
-            'institution': getattr(getattr(user, 'profile'), 'institution', '')
-        } for user in getattr(pub, 'users', [])]
+        if pub.__contains__('users'):
+            context['authors'] = [{
+                'full_name': '{last_name}, {first_name}'.format(
+                    last_name=user['last_name'].encode('utf-8'), first_name=user['first_name'].encode('utf-8')
+                ),
+                'institution': getattr(getattr(user, 'profile'), 'institution', '')
```

Comment:
Does writing this as `user.profile.institution` work?

Reply:
I used `getattr` because without it, if user.profile.institution is undefined, an AttributeError exception is thrown.

Reply:
I added the default parameter to this line in a new commit because without it, this function also throws an AttributeError.

Reply:
Thanks, I think that should work!